openslides/core/static/js/core/base.js

```diff
@@ -476,6 +476,12 @@ angular.module('OpenSlidesApp.core', [
 ])
 
 // Template hooks
+// 2 possible uses:
+// - { Id: 'myHookId', template: '<button>click me</button>' }
+// - { Id: 'myHookId', templateUrl: '/static/templates/plugin_name/my-hook.html' }
```

Comment:
I wonder whether there exists an AngularJS intern lib function to do what we want to do here: lazy loading of templates. If you don't have any idea just use this impelementation. It is ok for me.